### PR TITLE
e2e/tests/testdata: Bump from d980578d2e9b to b87e7c278283

### PR DESCRIPTION
**Pull request by wking**
In preparation for consuming the 'version' file that graph-data grew in openshift/cincinnati-graph-data#233, as ground-work for #314.  This gets us the first main-line commit to include the `version` file:

```console
cincinnati-graph-data$ git --no-pager show --date=short --format='%ad %h %s' d980578d2e b87e7c2782
2020-03-18 d980578 Merge pull request #125 from marun/unique-service-ca-serial

2020-05-11 b87e7c2 Merge pull request #233 from wking/version-file
```

[1]: https://github.com/openshift/cincinnati-graph-data/pull/233

### CODE REVIEW

File: e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.3_amd64-production.json
Comment (steveej) [inline e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.3_amd64-production.json]: This doesn't match https://github.com/openshift/cincinnati-graph-data/blob/b87e7c2782837a3538b0aa3dceb9e8133765ec81/blocked-edges/4.3.0.yaml#L2

**Review by steveej — CHANGES_REQUESTED** [review]
I could explain the stable-4.3_amd64 with an inconsistency in the test fixtures.
I can't do the same [stable-4.2_s390x](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cincinnati/343/pull-ci-openshift-cincinnati-master-e2e/1322253119068835840#1:build-log.txt%3A643) so I suspect a bug somewhere in the existing comparison code.

**Review by steveej — APPROVED** [review]
/lgtm

To prevent self-approving after pushing code to this PR myself I'll let someone else remove the hold.

/hold

**Comment by wking** [comment]
I've pushed e2287121 -> 3c1185a0, adding `hack/graph-normalize.sh` to get stable ordering and washing the pre- and post-bump data through that script.  This makes the diff slightly cleaner (e.g. `git --no-pager show --oneline --minimal -M2 3c1185a0`).  There is no significant fixture diff vs. e2287121:

```console
$ git checkout e2287121
$ git checkout 3c1185a0 hack/graph-normalize.sh
$ for X in e2e/tests/testdata/*.json; do A="$(hack/graph-normalize.sh < "${X}")"; echo "${A}" > "${X}"; done
$ git --no-pager diff 3c1185a0 -- e2e/tests/testdata
diff --git a/e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.2_amd64-production.json b/e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.2_amd64.json
similarity index 100%
rename from e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.2_amd64-production.json
rename to e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.2_amd64.json
diff --git a/e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.2_s390x-production.json b/e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.2_s390x.json
similarity index 100%
rename from e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.2_s390x-production.json
rename to e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.2_s390x.json
diff --git a/e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.3_amd64-production.json b/e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.3_amd64.json
similarity index 100%
rename from e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.3_amd64-production.json
rename to e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.3_amd64.json
diff --git a/e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.3_s390x-production.json b/e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.3_s390x.json
similarity index 100%
rename from e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.3_s390x-production.json
rename to e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.3_s390x.json
```

**Comment by steveej** [comment]
e2e tests will fail until you add the change in this line: 

https://github.com/openshift/cincinnati/commit/e2287121eb1d4591adf4ef44b081571de9c6085a#diff-9bac4b20dd90ff48de848f78e8a77322d954e17bff639022c7ebb01c4d6ad807R17

**Review by steveej — APPROVED** [review]
/lgtm

**Comment by wking** [comment]
Hmm.  This was pulled in when #314, which built on top, passed its CI.  But [the e2e run here][1] crashed out with:

```
test check_slo::_cincinnati_gb_graph_upstream_errors_total_0_ ... FAILED
test check_slo::_max_over_time_cincinnati_gb_graph_upstream_errors_total_1h_0_ ... FAILED
failures:
---- check_slo::_cincinnati_gb_graph_upstream_errors_total_0_ stdout ----
thread 'check_slo::_cincinnati_gb_graph_upstream_errors_total_0_' panicked at 'assertion failed: `(left == right)`
  left: `"1"`,
 right: `"0"`', e2e/tests/slo.rs:55:5
stack backtrace:
   0: backtrace::backtrace::libunwind::trace
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.46/src/backtrace/libunwind.rs:86
  ...
  13: std::panicking::begin_panic_fmt
             at src/libstd/panicking.rs:391
  14: slo::check_slo::check_slo
             at e2e/tests/slo.rs:55
  15: slo::check_slo::_cincinnati_gb_graph_upstream_errors_total_0_
             at e2e/tests/slo.rs:7
  16: slo::check_slo::_cincinnati_gb_graph_upstream_errors_total_0_::{{closure}}
             at e2e/tests/slo.rs:7
```

I don't see how a fixture bump could have caused a panic in `e2e/tests/slo`, but maybe a pre-existing issue?

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cincinnati/343/pull-ci-openshift-cincinnati-master-e2e/1324432371671371776#1:build-log.txt%3A307

**Pull request merged**
